### PR TITLE
nsxiv: Add at v32

### PR DESCRIPTION
**Pull request by androidnisse**
**Summary**
- nsxiv: Add at v32
- Replaces sxiv

**Test Plan**
- Installed, opened an image, read the man pages.

**Checklist**

- [X] Package was built and tested against unstable

**Packaging notes**
- Resolves https://github.com/getsolus/packages/issues/3007

### CODE REVIEW

**Review by malfisya — CHANGES_REQUESTED** [review]
A few changes:
- Let's use a better comit message:
```
package: Add at v1.0
```
- This package include a desktop file but doesn't ship an appstream metainfo file. Can you add that? There is an example in this [issue](https://github.com/getsolus/packages/issues/1389). I usually use this [tool](https://www.freedesktop.org/software/appstream/metainfocreator/#/guiapp) to create appstream metainfo.

**Comment by androidnisse** [comment]
> A few changes:
> 
>     * Let's use a better comit message:
> 
> 
> ```
> package: Add at v1.0
> ```
> 
>     * This package include a desktop file but doesn't ship an appstream metainfo file. Can you add that? There is an example in this [issue](https://github.com/getsolus/packages/issues/1389). I usually use this [tool](https://www.freedesktop.org/software/appstream/metainfocreator/#/guiapp) to create appstream metainfo.

Does it work like I did now?

File: packages/n/nsxiv/package.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+source     :
+    - https://codeberg.org/nsxiv/nsxiv/archive/v32.tar.gz : 09d1d72b3cbcf17a04e26beb5e81acc9495aaba1f8f1be907bdcd8e4e3007db3
+homepage   : https://nsxiv.codeberg.page/
+license    : GPL-2.0-or-later
+component  : multimedia.graphics
+summary    : Neo (or New or Not) Simple (or Small or Suckless) X Image Viewer.
+description: |
+    Nsxiv is a fork of the now-unmaintained sxiv with the purpose of being a (mostly) drop-in replacement for sxiv, maintaining its interface and adding simple, sensible features.
+replaces   :
+    - sxiv
```
Comment (malfisya) [inline packages/n/nsxiv/package.yml:13]: Lets put this at the bottom

**Review by malfisya — CHANGES_REQUESTED** [review]
Yes, the appstream metainfo is valid. I just noticed this one small thing, other than that it looks good. Cheers!

**Review by malfisya — APPROVED** [review]
Looks good. Thanks!

**Comment by malfisya** [comment]
Oh, do you want to include `sxiv` deprecation here or file seperate PR? We might as well do it soon.

**Comment by androidnisse** [comment]
> Oh, do you want to include `sxiv` deprecation here or file seperate PR? We might as well do it soon.

I can file another soon for deprecation.

**Pull request merged**